HW_4/asynchronous/asynchronous.py

```diff
@@ -0,0 +1,37 @@
+# Асинхронный
+
+import time
+import asyncio
+import aiohttp
+from sys import argv
+
+start_time = time.time()
+all_time = 0
+urls = argv[1::]
```

Comment:
Если шаг не передается, то можно делать так
```suggestion
urls = argv[1:]
```